webapp/app/js/filters.js

```diff
@@ -39,11 +39,21 @@ treeherder.filter('stripHtml', function() {
 
 treeherder.filter('linkifyBugs', function() {
     return function(input) {
-        var re = new RegExp('(?:Bug (\\d+))', 'ig');
-        var str = input || '';
-        return str.replace(re,
-            '<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=$1" target="_blank" title="Bug $1">$1</a>'
-        );
+        if (input) {
```

Comment:
Ok, the logic is good now, however I think I still prefer the earlier way this was implemented, where we created a variable str equal to '' if input was not defined, and just returned that (var str = input || ''). That removes the need for if..else.